src/js/services/conversation.js

```diff
@@ -15,6 +15,59 @@ import { hasLinkableChannels, getLinkableChannels, isChannelLinked } from '../ut
 import { CONNECT_NOTIFICATION_DELAY_IN_SECONDS } from '../constants/notifications';
 import { getUserId } from './user';
 
+const postSendMessage = (dispatch, getState, message) => {
```

Comment:
these functions are actions, they shouldn't be here. 

You can certainly move them in the conversation actions and transform them as 
```
function postSendMessage(message) {
    return (dispatch, getState) => { ... };
};
```
instead.